### ISSUE TITLE
Add permission to all roles by default

### ISSUE THREAD

**Issue by ghost** [backdrop-contrib/pwa#11]
I initially had trouble getting my app to work, until I realised there was a permission that was only assigned to the Anonymous role by default (and I was logged in).

Recommend assigning the 'access pwa' permission to all roles by default.

**Comment by swampopus** [comment]
I agree.  Fixed in release https://github.com/backdrop-contrib/pwa/releases/tag/1.x-1.0.2
